main/static/style.css

```diff
@@ -13,22 +13,16 @@ body {
 }
 
 footer{
-    padding-top: 30px;
     margin: auto;
-    align: center;
-    /*border-top-color: #5b80b2;*/
-    /*border-top-width: thick;*/
+    text-align: center;
 }
 
-#headerimg{
+.pingimg{
     width:128px;
-    height:128px;"
-}
-
-#footergif{
-    width: 70%;
+    height:128px;
 }
 
-#content{
-    /*background: blue;*/
+img:hover {
+    -webkit-transform: scaleX(-1);
+    transform: scaleX(-1);
 }
```

Comment:
Minor: please check the spacing of this file.
It's also recommended to always add a trailing newline - [read a bit about why that is](https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline).